examples/resources/ekscluster/cluster.tf

```diff
@@ -50,19 +50,16 @@ resource "tanzu-mission-control_ekscluster" "tf_eks_cluster" {
         ]
       }
 
-      addons_config {
+      addons_config { // this whole section is optional
         vpc_cni_config {
           eni_config {
-            id = "subnet-0a680171b6330619f" // Required, should belong to the same VPC as the cluster
-            security_groups = [
+            id = "subnet-0a680171b6330619f" // Required, should belong to the same VPC as the cluster, subnets provided in vpc_cni_config are expected to be in different AZs
+            security_groups = [ //optional
               "sg-00c96ad9d02a22522",
             ]
           }
           eni_config {
```

Comment:
good to point out that the EKS cluster security group will be used here

Reply:
@DheerajSShetty fixed this